mkdocs-en.yml

```diff
@@ -1,110 +1,557 @@
+            - Cluster Insight: zh/insight/06UserGuide/03scenarioinsight/clusterinsight.md
+            - Node Insight: zh/insight/06UserGuide/03scenarioinsight/nodeinsight.md
+            - Container Insight: zh/insight/06UserGuide/03scenarioinsight/containerinsight.md
+            - Service Insight: zh/insight/06UserGuide/03scenarioinsight/serviceinsight.md
+          - Data Query:
+            - Log Query: zh/insight/06UserGuide/04dataquery/logquery.md
+            - Metric Query: zh/insight/06UserGuide/04dataquery/metricquery.md
+            - Trace Query: zh/insight/06UserGuide/04dataquery/tracequery.md
+          - Alert Center:
+            - Alert Rules: zh/insight/06UserGuide/05alertcenter/alertrule.md
+            - Notifications: zh/insight/06UserGuide/05alertcenter/message.md
+            - Message Template: zh/insight/06UserGuide/05alertcenter/messagetemplate.md
+          - Data Collection:
+            - Collection Management: zh/insight/06UserGuide/07collectionmanag/collectionmanag.md
+            - Service Insight: zh/insight/06UserGuide/07collectionmanag/servicemonitor.md
+            - Enable Collection Log: zh/insight/09FAQs/collectauditlog.md
+          - System Configuration:
+            - System Configuration: zh/insight/06UserGuide/08systemconfig/systemconfig.md
+            - Modify Configuration: zh/insight/06UserGuide/08systemconfig/modifyconfig.md
+          - FAQs: 
+            - zh/insight/09FAQs/README.md
+            - Collection Log: zh/insight/09FAQs/collectauditlog.md
+      - App Workbench:
+        - Introduction:
+          - What is App Workbench: zh/amamba/01ProductBrief/WhatisAmamba.md
+          - Features: zh/amamba/01ProductBrief/Features.md
+          - Benefits: zh/amamba/01ProductBrief/Benefits.md
+          - Scenarios: zh/amamba/01ProductBrief/Scenarios.md
+          - Concepts: zh/amamba/01ProductBrief/Concepts.md
+        - Create Pipeline: zh/amamba/02QuickStart/deploypipline.md
+        - Pipelines:
+          - Custom Pipeline: zh/amamba/03UserGuide/Pipeline/createpipelinebyself.md
+          - Create Pipeline with Jenkinsfile: zh/amamba/03UserGuide/Pipeline/createpipelinebyjenkins.md
+          - Edit Pipeline with GUI: zh/amamba/03UserGuide/Pipeline/graphicaleditingpipeline.md
+          - Credentials: zh/amamba/03UserGuide/Pipeline/Credential.md
+          - Pipeline Parameters:
+            - Pipeline Concepts: zh/amamba/03UserGuide/Pipeline/PipelineConfig/PipelineConcept.md
+            - Job Template Parameter with GUI: zh/amamba/03UserGuide/Pipeline/PipelineConfig/PipelineStep.md
+      - What is Multi-Cloud Orchestration:
+        - Introduction:
+          - What is Multi-Cloud Orchestration: zh/kairship/01product/whatiskair.md
+          - Features: zh/kairship/01product/features.md
+          - Benefits: zh/kairship/01product/benefits.md
+          - Scenarios: zh/kairship/01product/scenarios.md
+          - Concepts: zh/kairship/01product/concepts.md
+        - Installation:
```

Comment:
may be we should consistently use either "install" as in line 99 or "installation" as in this line.